values.yaml

```
@@ -45,7 +45,7 @@ global:
 
   # Enable default postgresql database.
   # This is not recommended for production.
-  postgresqlEnabled: false
+  postgresqlEnabled: true
```

This should not change.
```suggestion
  postgresqlEnabled: false
```